packages/run-protocol/src/vpool-xyk-amm/priceAuthority.js

```diff
@@ -66,24 +70,25 @@ export const makePriceAuthority = (
     actualBrandIn,
     actualBrandOut,
     quoteIssuer,
-    notifier,
+    notifier: makeNotifierFromSubscriber(subscriber),
   });
 
   const {
     priceAuthority,
     adminFacet: { fireTriggers },
   } = makeOnewayPriceAuthorityKit(priceAuthorityOptions);
 
-  // An observer on a notifier that gets updated every time the pool's balances
+  // An observer on a notifier that gets updated when the pool's balances
   // change via swap or liquidity operations. It checks the comparison predicate
-  // for each active trigger.
+  // for each active trigger. Due to `subscribeLatest` below it may drop
+  // intermediate values.
```

Comment:
We need to be able to get every pool balance change via RPC. Is that still available after this change?